backend/src/main/java/com/soen341/instagram/service/impl/ProfileService.java

```diff
@@ -32,103 +31,69 @@
 //		account.setUsername(username);
 //		accountRepository.save(account);
 //	}
-	
+
 //TODO: Implement changing profile pictures	
```

Comment:
Same comment as before: if this isn't feasible by the end of this sprint, better remove this block of code